jobs/JGLOBAL_ATMOS_UPP

```diff
@@ -12,8 +12,11 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "upp" -c "base upp"
 ##############################################
 
 # Construct COM variables from templates
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS COM_ATMOS_HISTORY COM_ATMOS_MASTER
-if [[ ! -d ${COM_ATMOS_MASTER} ]]; then mkdir -m 775 -p "${COM_ATMOS_MASTER}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
+   COMIN_ATMOS_HISTORY:COM_ATMOS_HISTORY \
+   COMOUT_ATMOS_MASTER:COM_ATMOS_MASTER
+if [[ ! -d ${COMOUT_ATMOS_MASTER} ]]; then mkdir -m 775 -p "${COMOUT_ATMOS_MASTER}"; fi
```

Comment:
```suggestion
if [[ ! -d ${COMOUT_ATMOS_MASTER} ]]; then mkdir -p "${COMOUT_ATMOS_MASTER}"; fi
```
Please remove `-m 775`

Reply:
I have removed "-m 775" on line 19.

Reply:
@mingshichen-noaa 
I still see `-m 775` in this PR.  Did you push your change?

Reply:
@aerorahul 
After I removed "-m 775" and then git add  file, git commit -m ""-m 775" , 
finally git push origin HEAD:feature/atmos_upp_com_in_out.
Just now I tried to do the above mentioned final step again. 
But after I checked jobs/JGLOBAL_ATMOS_UPP in the PR, "-m 775" is still over there. So far I have not identified the cause.

Reply:
@aerorahul 
I identified the reason why you see  still "-m 775 in this PR". This is because I created two branches: origin/atmos_upp_com_in_out and  origin/feature/atmos_upp_com_in_out. I did wrongly git push the modified files into origin/atmos_upp_com_in_out.  in order to avoid the mistakes again, I have deleted the origin/atmos_upp_com_in_out.
Please review the PR again. Thanks.